src/SWP/Bundle/ContentBundle/Resources/config/services.yml

```diff
@@ -107,3 +108,27 @@ services:
         parent: "router.symfony"
         tags:
             - { name: router, priority: 300 }
+
+    swp_content_bundle.valid_route_id:
+        class: SWP\Bundle\ContentBundle\Validator\Constraints\RouteIdValidator
+        arguments:
+            - '@swp.provider.route'
+        tags:
+            - { name: validator.constraint_validator, alias: route_in_database }
+
+    swp_content_bundle.routetoarticle.repository:
```

Comment:
OK, this is changed in the latest commit. I must say that after making the change, I can't really understand the advantage of it. The configuration seemed simpler before and could still be overridden. Anyway, there it is.